### PR TITLE
improve personal voice samples

### PR DESCRIPTION
**Pull request by szhaomsft**
## Purpose
make the sample easier to run 

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

**Review by ms-feizhao — APPROVED** [review]

File: samples/custom-voice/python/customvoice/config.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 import logging
 
 class Config:
-    api_version = 'api-version=2023-12-01-preview'
+    api_version = 'api-version=2024-02-01-preview'
```
Comment (ForrestGumb) [inline samples/custom-voice/python/customvoice/config.py:10]: This version is in the progress of release. Please merge after release, or revert this change. We'll update this line after release.

File: samples/custom-voice/python/voice/VoiceTalentVerbalStatement.wav
Comment (ForrestGumb) [inline samples/custom-voice/python/voice/VoiceTalentVerbalStatement.wav:1]: Please put this legal doc beside this sample.
https://github.com/Azure-Samples/Cognitive-Speech-TTS/blob/master/CustomVoice/Sample%20Data/License-MustRead.md

**Review by ForrestGumb — APPROVED** [review]

**Review by yulin-li — APPROVED** [review]
